draft-ietf-ppm-dap.md

```diff
@@ -988,10 +988,10 @@ report for this reason, it SHOULD abort the upload protocol and alert the Client
 with error `reportTooEarly`. In this situation, the Client MAY re-upload the
 report later on.
 
-If the Leader's ReportShare contains an unrecognized extension, or if two
+If the Leader's input share contains an unrecognized extension, or if two
```

Comment:
This should say "report share" instead of "input share". The latter is the thing consumed by VDAF; the former is the thing consumed by DAP. We need to make this distinction clear, somehow, and use terminology consistently. (Here and below.)

Reply:
Actually it looks like there are multiple places where this regression happened.

Reply:
I think that's incorrect: this section has no concept of report shares (which are not introduced until the aggregation sections), and indeed now report shares are gone entirely (as the `ReportShare` message was merged into the new `PrepareInit` message, since that was the only usage of the `ReportShare` message). Specifically, the message that would contain the extensions referred to in this text is called `PlaintextInputShare`.

IMO, if we want to change terminology, we should do so in a later PR that updates all usages of "input share" in this section; but for now, this textual change is a simple bugfix.

Reply:
Extensions occur in `struct PlaintextInputShare` (a DAP-level concept), so this is more accurate than `ReportShare.

Reply:
From the glossary section:
```
Report:
: A cryptographically protected measurement uploaded to the Leader by a Client.
  Comprised of a set of report shares.

Report Share:
: An encrypted input share comprising a piece of a report.
```

Reply:
Conceptually a "report share" is everything an Aggregator needs in order to aggregate its share of the Client's input. Semantically this ought to include the report extensions.

Reply:
FWIW, it seems that there is some terminology confusion/disagreement around the difference between an "input share" and a "report share"; I suggest we discuss outside this PR, come to consensus, and fix things up throughout the spec in a later PR.

Reply:
I agree. Any ambiguity between input and report shares existed before this change so we shouldn't try to solve that problem here.

Reply:
Agreed, let's make sure we get some clarity on this. But does it make sense to at least revert the change of replacing "ReportShare" to "input share" in this PR?